eng/pipelines/mono/templates/xplat-pipeline-job.yml

```diff
@@ -0,0 +1,67 @@
+parameters:
```

Comment:
Instead of introducing another template, can we use coreclr's and move it to common?

Reply:
I tried to share them but there were too many differences in paths which weren't easy to fix. We should file an issue to consolidate runtime infrastructure files where possible (not just yml).

Reply:
Sounds good. Let's please open that issue as it seems overkill to have this duplicate templates (maintaining multiple yml files is painful).

Reply:
https://github.com/dotnet/runtime/issues/2029